shared/wallets/transaction-details/container.js

```diff
@@ -73,6 +73,7 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     title: 'Transaction details',
     transactionID: tx.txID,
     you: stateProps.you,
+    youWereWallet: tx.sourceType ? tx.source : '',
```

Comment:
`tx.sourceType === 'otherAccount'`

Reply:
actually 'ownaccount'

Reply:
whoops, sorry

Reply:
Wait, we have `ownAccount` _and_ `otherAccount`? 🤔